build/config/index.js

```diff
@@ -0,0 +1,18 @@
+const config = {
+  production: {
+    APP_PORT: '4001',
+    APP_SECRET: 'someappsecrethere',
```

Comment:
Put all these things in the .env file and access it from `process.env`.

Reply:
Done as requested.